lstchain/scripts/lstchain_tune_nsb_waveform.py

```diff
@@ -88,7 +89,7 @@ def main():
     dict_nsb = {
         "nsb_tuning": True,
         "nsb_tuning_ratio": np.round(nsb_correction_ratio, decimals=2),
-        "spe_location": "lstchain/data/SinglePhE_ResponseInPhE_expo2Gaus.dat"
+        "spe_location": get_resource_path("data/SinglePhE_ResponseInPhE_expo2Gaus.dat")
```

Comment:
`get_reource_path` returns a Path object, I do not know if that'd be a problem here

Reply:
No problem there, it's then passed to `loadtxt`